app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)

```

Comment:
Code Style: It's a good practice to use parentheses `()` while importing multiple classes/modules. You've done it correctly here, keep it up!